Source/WebCore/platform/graphics/ca/GraphicsLayerCA.cpp

```diff
@@ -515,7 +515,7 @@ String GraphicsLayerCA::debugName() const
     String caLayerDescription;
     if (!m_layer->isPlatformCALayerRemote())
         caLayerDescription = makeString("CALayer(0x", hex(reinterpret_cast<uintptr_t>(m_layer->platformLayer()), Lowercase), ") ");
-    return makeString(caLayerDescription, "GraphicsLayer(0x", hex(reinterpret_cast<uintptr_t>(this), Lowercase), ", ", primaryLayerID(), ") ", name());
+    return makeString(caLayerDescription, "GraphicsLayer(0x", hex(reinterpret_cast<uintptr_t>(this), Lowercase), ", ", primaryLayerID().toUInt64(), ") ", name());
```

Comment:
I wonder if we can teach `makeString()` how to handle `ObjectIdentifier<>`?

Reply:
We can, but should we?

Reply:
https://github.com/WebKit/WebKit/pull/8557